### DISCUSSION TITLE
Replace yarn in docs

### DISCUSSION THREAD

**Discussion by benmccann** [vuejs/vitepress#1149] [General]
### Describe the bug

I just went to the getting started page and it tells me to use yarn. That seems like a weird choice since npm is a lot more popular and this repo uses pnpm. Either of those would make more sense to me

### Reproduction

https://vitepress.vuejs.org/guide/getting-started.html

### Expected behavior

I expect to see npm or maybe pnpm

### System Info

```shell
n/a
```

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.vuejs.org).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by brc-dd** [comment]
My __*personal*__ opinions on this:

- pnpm and yarn berry still require additional config to work (peer deps issue), so not gonna write those on docs everywhere. Then the game is good old npm vs yarn classic.
- Personally I believe that yarn is much better than npm (in terms of stuff like performance)  and is fairly popular too (3M vs 4M downloads/week and 40K vs 17+6K stars).
- Also, it's there because no one cared to modify it. We had that in VuePress v1 because of some issue with webpack, VuePress v2 and VitePress sort of went along with that.

We can change that if people want so, I don't have any hard opinions about that.
